### PR TITLE
*: update docs-dm

### PR DESCRIPTION
**Pull request by Ehco1996**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

add shortcuts to generate dm template

### Which DM version(s) do your changes apply to? (Required)
- [x] master (the latest development version, including v3.0 changes)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by CharLotteiu** [comment]
Suggest changing the title to `[WIP] *: add shortcuts to generate dm template`. 😄

**Comment by Ehco1996** [comment]
this PR is ready, plz cc @CharLotteiu

**Review by lance6716 — APPROVED** [review]

**Comment by CharLotteiu** [comment]
Do the changes in this PR need to be cherry picked to the release-2.0 branch? @Ehco1996

**Comment by Ehco1996** [comment]
> Do the changes in this PR need to be cherry picked to the release-2.0 branch? @Ehco1996

yes, i think so

File: zh/deploy-a-dm-cluster-using-binary.md [hunk 1]
```diff
@@ -129,6 +129,10 @@ initial-cluster = "master1=http://192.168.0.4:8291,master2=http://192.168.0.5:82
 
 {{< copyable "shell-regular" >}}
 
+> **注意：**
+>
+> 执行该命令后控制台不会输出日志，可以通过 `tail -f dm-master.log` 查看运行日志
```
Comment (CharLotteiu) [inline zh/deploy-a-dm-cluster-using-binary.md:134]:
```suggestion
> 执行该命令后控制台不会输出日志，可以通过 `tail -f dm-master.log` 查看运行日志。
```

File: en/deploy-a-dm-cluster-using-binary.md [hunk 2]
```diff
@@ -134,6 +134,10 @@ The following is the configuration file of DM-master. It is recommended that you
       ./bin/dm-master -config conf/dm-master1.toml
       ```
 
+      > **Note:**
+      >
+      > After executing this command, the console will not output the log, you can view the running log through `tail -f dm-master.log`
```
Comment (CharLotteiu) [inline en/deploy-a-dm-cluster-using-binary.md:139]:
```suggestion
      > The console does not output logs after this command is executed. If you want to view the runtime log, you can execute `tail -f dm-master.log`.
```

File: zh/deploy-a-dm-cluster-using-tiup.md [hunk 3]
```diff
@@ -68,6 +68,8 @@ aliases: ['/docs-cn/tidb-data-migration/dev/deploy-a-dm-cluster-using-ansible/',
 
 请根据[配置文件模板](https://github.com/pingcap/tiup/blob/master/embed/templates/examples/dm/topology.example.yaml)，新建一个配置文件 `topology.yaml`。如果有其他组合场景的需求，请根据多个模板自行调整。
 
+> 可以通过 `tiup dm template > topology.yaml` 命令快速生成模板
```
Comment (CharLotteiu) [inline zh/deploy-a-dm-cluster-using-tiup.md:71]:
```suggestion
可以使用 `tiup dm template > topology.yaml` 命令快速生成配置文件模板。
```

File: en/deploy-a-dm-cluster-using-tiup.md [hunk 4]
```diff
@@ -68,6 +68,8 @@ According to the intended cluster topology, you need to manually create and edit
 
 You need to create a YAML configuration file (named `topology.yaml` for example) according to the [configuration file template](https://github.com/pingcap/tiup/blob/master/embed/templates/examples/dm/topology.example.yaml). For other scenarios, edit the configuration accordingly.
 
+> You can use `tiup dm template > topology.yaml` command to generate template quickly
```
Comment (CharLotteiu) [inline en/deploy-a-dm-cluster-using-tiup.md:71]:
```suggestion
You can use the command `tiup dm template > topology.yaml` to generate a configuration file template quickly.
```

File: zh/quick-start-with-dm.md [hunk 5]
```diff
@@ -266,3 +266,31 @@ block-allow-list:
 ```
 
 这样就成功创建了一个将 MySQL-3306 数据迁移到 TiDB 的任务。
+
+### 查看迁移任务状态
+
+在创建迁移任务之后，可以用 `dmtcl query-status` 来查看任务的状态
```
Comment (CharLotteiu) [inline zh/quick-start-with-dm.md:272]:
```suggestion
在创建迁移任务之后，可以用 `dmtcl query-status` 来查看任务的状态。
```

**Review by CharLotteiu — COMMENTED** [review]
rest LGTM

**Review by CharLotteiu — APPROVED** [review]
LGTM

**Pull request merged**